### PR TITLE
Support symbolink link

### PR DESCRIPTION
**Pull request by rubensa**
Fixes: https://github.com/angelozerr/angular2-eclipse/issues/92

### CODE REVIEW

File: core/ts.core/src/ts/utils/ZipUtils.java [hunk 1]
```diff
@@ -172,12 +173,22 @@ public static void extractTar(File file, File destination) throws IOException {
 					break;
 				case TarEntry.LINK:
 					File linkFile = new File(destination, outFilename);
+					// Be sure that parent file exists
+					File linkBaseDir = linkFile.getParentFile();
+					if (!linkBaseDir.exists()) {
+						linkBaseDir.mkdirs();
+					}
 					Path target = new File(linkFile.getParentFile(), entry.getLinkName()).toPath();
```
Comment (angelozerr) [inline core/ts.core/src/ts/utils/ZipUtils.java]: Should be not Path target = Paths.get(entry.getLinkName()); ?

Comment (rubensa) [inline core/ts.core/src/ts/utils/ZipUtils.java]: Probably...  but as sample file didn't have a "link" (only "symlinks") I didn't changed that part.  
I'll try to test with new .tar.tz file that includes a link.

Comment (rubensa) [inline core/ts.core/src/ts/utils/ZipUtils.java]: Ummm...  seems that I can't "really" create a tar.gz file that contains "hard links" cause when packaged, the "hard link" is a copy of the original linked file (so, no information that it was a link).  Any way I'm going to modify that part of the code (but I think that is never been executed).

**Comment by angelozerr** [comment]
Thanks a lot @rubensa !

**Pull request merged**